docs/examples/nginx-services/README.md

```diff
@@ -0,0 +1,88 @@
+# Interlock with Docker Services
+Starting in Docker 1.12 there is support for Services.  In this example, we
+will use Services to run Interlock and the Proxy.
+
+Note: if you have not initialized the Swarm, make sure to init it with `docker swarm init`
+
+# Nginx
+This will create a global Nginx service that will run on every node and
+publish port 80:
+
+```
+docker service create \
+    --name interlock-nginx \
+    --publish 80:80 \
+    --mode global \
+    --label interlock.ext.name=nginx \
+    nginx \
+        nginx -g "daemon off;" -c /etc/nginx/nginx.conf
+```
+
+You should now be able to visit `http://<node-ip>` and see the Nginx welcome
+page.
+
+# Interlock
+We will now start a global Interlock service that will run on every node:
+
+First, we need an Interlock configuration.  Create a file called `config.toml`
+with the content:
+
+```
+ListenAddr = ":8080"
+DockerURL = "unix:///var/run/docker.sock"
+PollInterval = "2s"
+
+[[Extensions]]
+Name = "nginx"
+ConfigPath = "/etc/nginx/nginx.conf"
+PidPath = "/var/run/nginx.pid"
+TemplatePath = ""
+MaxConn = 1024
+Port = 80
+```
+
+Now create the Interlock service:
+
+```
+docker service create \
+    --mode global \
+    --name interlock \
+    --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock,writable=true \
```

Comment:
According to https://github.com/docker/docker/pull/24053
`writable=true` is the default behavior and the flag has been removed